hbs-helpers.js

```diff
@@ -42,13 +42,27 @@ function localizedBreachDataClasses(supportedLocales, dataClasses, args) {
 
 
 function fluentNestedBold(supportedLocales, id, args) {
-  if (args.hash.breachName) {
-    args.hash.breachName = `<span class="medium">${args.hash.breachName}</span>`;
-  }
+  const saveArgs = JSON.parse(JSON.stringify(args.hash));
```

Comment:
Hmm ... not sure why we're calling `JSON.parse(JSON.stringify(...))`? Isn't that a no-op?

Reply:
It's generally a cheap way to clone an object so it doesn't modify the original `args.hash` object.